htmd/queues/lsfqueue.py

```diff
@@ -23,8 +23,8 @@ class LsfQueue(SimQueue, ProtocolInterface):
     ----------
     jobname : str, default=None
         Job name (identifier)
-    queue : str, default=None
-        The queue to run on
+    queue : str or list of str, default=None
+        The queue or list of queues to run on. If list, it attempts to submit the job to the first queue listed
```

Comment:
And then? It doesn't use the rest? Or it falls back to the others in order?

Reply:
Yeah, it goes on the others. I can complete the documentation.